plugins/woocommerce/src/Internal/Admin/Reviews.php

```diff
@@ -0,0 +1,603 @@
+			wp_die( esc_html( $updated->get_error_message() ) );
+		}
+
+		$position = isset( $_POST['position'] ) ? (int) sanitize_text_field( wp_unslash( $_POST['position'] ) ) : -1;
+		$wp_list_table = $this->make_reviews_list_table();
+
+		ob_start();
+		$wp_list_table->single_row( $review );
+		$review_list_item = ob_get_clean();
+
+		$x = new WP_Ajax_Response();
+
+		$x->add(
+			array(
+				'what'     => 'edit_comment',
+				'id'       => $review->comment_ID,
+				'data'     => $review_list_item,
+				'position' => $position,
+			)
+		);
+
+		$x->send();
+	}
+
+	/**
+	 * Ajax callback for replying to a review inline.
+	 *
+	 * This functionality is taken from {@see wp_ajax_replyto_comment()} and is largely copy and pasted. The only thing
+	 * we want to change is the review row HTML in the response. WordPress core uses a comment list table and we need
+	 * to use our own {@see ReviewsListTable} class to support our custom columns.
+	 *
+	 * This ajax callback is registered with a lower priority than WordPress core's so that our code can run
+	 * first. If the supplied comment ID is not a review or a reply to a review, then we `return` early from this method
+	 * to allow the WordPress core callback to take over.
+	 *
+	 * @return void
+	 */
+	public function handle_reply_to_review(): void {
+		check_ajax_referer( 'replyto-comment', '_ajax_nonce-replyto-comment' );
+
+		$comment_post_ID = isset( $_POST['comment_post_ID'] ) ? (int) sanitize_text_field( wp_unslash( $_POST['comment_post_ID'] ) ) : 0; // phpcs:ignore WordPress.NamingConventions.ValidVariableName.VariableNotSnakeCase
+		$post            = get_post( $comment_post_ID ); // phpcs:ignore WordPress.NamingConventions.ValidVariableName.VariableNotSnakeCase
+
+		if ( ! $post ) {
+			wp_die( -1 );
+		}
+
+		// Inline Review replies will use the `detail` mode. If that's not what we have, then let WordPress core take over.
+		if ( isset( $_REQUEST['mode'] ) && 'dashboard' === $_REQUEST['mode'] ) {
+			return;
+		}
+
+		// If this is not a a reply to a review, bail silently to let WordPress core take over.
+		if ( 'product' !== get_post_type( $post ) ) {
+			return;
+		}
+
+		if ( ! current_user_can( 'edit_post', $comment_post_ID ) ) { // phpcs:ignore WordPress.NamingConventions.ValidVariableName.VariableNotSnakeCase
+			wp_die( -1 );
+		}
+
+		if ( empty( $post->post_status ) ) {
+			wp_die( 1 );
+		} elseif ( in_array( $post->post_status, array( 'draft', 'pending', 'trash' ), true ) ) {
+			wp_die( esc_html__( 'Error: You can\'t reply to a review on a draft product.', 'woocommerce' ) );
+		}
+
+		$user = wp_get_current_user();
+
+		if ( $user->exists() ) {
+			$user_ID              = $user->ID;
+			$comment_author       = wp_slash( $user->display_name );
+			$comment_author_email = wp_slash( $user->user_email );
+			$comment_author_url   = wp_slash( $user->user_url );
+			// WordPress core already sanitizes `content` during the `pre_comment_content` hook, which is why it's not needed here.
```

Comment:
```suggestion
			// WordPress core already sanitizes `content` during the `pre_comment_content` hook, which is why it's not needed here.  {@see wp_filter_comment()} and {@see kses_init_filters()}
```

Just to note exactly where the filters are applied.